README.md

```diff
@@ -21,13 +21,41 @@ This project aims to generate structured **PDF reports** from podcast interviews
    pip install -r requirements.txt
    ```
 
+Before running the application, ensure you have your `OPEN_API_KEY` and `GROK_API_KEY` configured.
```

Comment:
@cornzyblack @f-kuzey-edes-huyal do you need both of these keys or just either?

Reply:
@Sharonsyra so it should be either. If none are provided from the UI, then it looks in the environmental variables or the secrets.toml file